requirements.txt

```diff
@@ -11,6 +11,7 @@ azure-mgmt-monitor>=1.0.1
 azure-mgmt-network>=2.7.0
 azure-mgmt-resource>=2.2.0
 azure-mgmt-subscription>=0.2.0
+azure-storage-blob>=12.5.0
```

Comment:
I'm going to kick a few of these out of requirements.txt. This is ok to be optional with the other azure components right?

Reply:
yep thats good.